plugin/hotswap-agent-spring-plugin/src/main/java/org/hotswap/agent/plugin/spring/scanner/ClassPathBeanDefinitionScannerAgent.java

```diff
@@ -222,21 +215,10 @@ public BeanDefinition resolveBeanDefinition(byte[] bytes) throws IOException {
         Resource resource = new ByteArrayResource(bytes);
         resetCachingMetadataReaderFactoryCache();
         MetadataReader metadataReader = getMetadataReaderFactory().getMetadataReader(resource);
-        if (isCandidateComponent(metadataReader)) {
```

Comment:
Why is this check removed? I had to revert this change in #241, because otherwise all classes were regarded as Spring beans.